build ansible-runner via branch

emoruzzi opened the pull request.

emoruzzi committed:
build ansible-runner via branch

syndr commented:
Probably we should merge into `devel` with https://github.com/influxdata/awx-ee/pull/11 instead of this PR.

syndr committed:
Refactor release CI (#12)

* Refactor release CI

Image tags will be generated based upon the event that triggered the workflow:
  - push event:
    - main branch: latest
    - devel branch: devel
    - all other branches: <branch name>-latest
      Branches must be added to the on.push.branches array above to trigger
  - pull_request event: DEV-PR-<pull_request_number>
    Branches must be added to the on.pull_request.branches array above to trigger
  - release event: <tag_name>
  - schedule event: nightly
    This will be the same configuration as the 'latest' tag, but may contain updated packages, etc. from upstream
  - all other events: <first 7 chars of commit sha>

Run podman test as part of same workflow as release (fail faster)

* Build on PR to feature branch

* Update PR CI

No need to run on changes of only markdown files

emoruzzi commented:
> Probably we should merge into `devel` with #11 instead of this PR.

Yup - when I first opened, devel didn't exist :-D
